src/hotspot/share/gc/g1/g1RemSetSummary.cpp

```diff
@@ -44,7 +44,7 @@ void G1RemSetSummary::update() {
     CollectData(G1RemSetSummary * summary) : _summary(summary),  _counter(0) {}
     virtual void do_thread(Thread* t) {
       G1ConcurrentRefineThread* crt = static_cast<G1ConcurrentRefineThread*>(t);
-      _summary->set_rs_thread_vtime(_counter, crt->vtime_accum());
+      _summary->set_rs_thread_vtime(_counter, crt->cpu_time_s());
```

Comment:
There's still a bunch of "vtime" nomenclature in this file. Is the plan to clean that up later?

Reply:
I originally wanted to clean this up later, but added it now. There is some additional cleanup about the `HRRSIter` class, I filed an issue for that.